contracts/protocol/integration/auction-price/BoundedStepwiseLogarithmicPriceAdapter.sol

```diff
@@ -70,7 +70,7 @@ contract BoundedStepwiseLogarithmicPriceAdapter {
         if (lnExpression > type(uint256).max / scalingFactor) {
             return _getBoundaryPrice(isDecreasing, maxPrice, minPrice);
         }
-        uint256 priceChange = scalingFactor * lnExpression;
+        uint256 priceChange = FixedPointMathLib.mulWad(scalingFactor, lnExpression);
```

Comment:
Add a test case for the case that this code protects against - i.e. moving from higher dp to lower dp.

Reply:
Will add WETH -> USDC and USDC -> WETH examples